### DISCUSSION TITLE
skyView driver cannot be used in osgearth3.2

### DISCUSSION THREAD

**Discussion by cxhaizxm** [pelicanmapping/osgearth#1793] [Support Q&A]
I run osgearth_skyview and osgearth_viewer to view the skyview1.earth. Get error information. I examine the bin plugins folder,do not find the skyview plugin. How to let the plugin use succefully. Thank you very much.

the earth is used above.
 <Map name="SkyView Demo">
    
    <options>
        <terrain>
            <cluster_culling>false</cluster_culling>
            <skirt_ratio>0.0</skirt_ratio>
        </terrain>
    </options>
   
    
    <SkyView>
        <GDALImage name="EarthAtNight">
	        <url>../data/skyview/celestial-ref.jpg</url>
	        <profile>global-geodetic</profile>
	    </GDALImage>
    </SkyView>
</Map>

![image](https://user-images.githubusercontent.com/7930275/133233585-83006bd0-8934-4ab3-bb78-48c66f7753a9.png)

![image](https://user-images.githubusercontent.com/7930275/133232665-f971a3b7-c482-41b7-a5a0-370a65b754b0.png)

**Comment by cxhaizxm** [comment]
the earth file is used above.
![image](https://user-images.githubusercontent.com/7930275/133234732-38617ee9-55d7-4e74-85ac-8457158596d4.png)

**Comment by gwaldron** [comment]
There is no skyview plugin in osgEarth 3.2 -- it is just a normal layer built into the core library.

My output of `osgearth_skyview skyview1.earth` looks like this (master branch):

![image](https://user-images.githubusercontent.com/326618/133285649-194e75fe-c926-4895-a81c-583ea6d04142.png)
